src/host/buildvm.c

```diff
@@ -86,7 +86,8 @@ static void emit_raw(BuildCtx *ctx) {
 static const char *sym_decorate(BuildCtx *ctx, const char *prefix, const char *suffix) {
   char name[256];
   char *p;
-  sprintf(name, "%s%s", prefix, suffix);
+  const char *symprefix = ctx->mode == BUILD_machasm ? "_" : "";
```

Comment:
I'd add parentheses for readability:

```c
const char *symprefix = (ctx->mode == BUILD_machasm) ? "_" : "";
```

Reply:
I see your point, but I'd like to keep divergence from LuaJIT minimal in this part of the codebase. Pretty sure that this fragment will be a "write and forget" thing for a long time.